src/Controls/src/Core/Shell/Shell.cs

```diff
@@ -209,94 +377,216 @@ static void OnFlyoutBehaviorChanged(BindableObject bindable, object oldValue, ob
 		public static Color GetTabBarForegroundColor(BindableObject obj) => (Color)obj.GetValue(TabBarForegroundColorProperty);
-		/// <include file="../../../docs/Microsoft.Maui.Controls/Shell.xml" path="//Member[@MemberName='SetTabBarForegroundColor']/Docs/*" />
+
+		/// <summary>
+		/// Sets the foreground color for the tab bar. 
+		/// If the property is unset, the ForegroundColor property value is used.
+		/// </summary>
+		/// <param name="obj">The object to which the color is set.</param>
+		/// <param name="value">The foreground color for the tab bar.</param>
 		public static void SetTabBarForegroundColor(BindableObject obj, Color value) => obj.SetValue(TabBarForegroundColorProperty, value);
 
-		/// <include file="../../../docs/Microsoft.Maui.Controls/Shell.xml" path="//Member[@MemberName='GetTabBarTitleColor']/Docs/*" />
+		/// <summary>
+		/// Gets the foreground color for the tab bar. 
+		/// </summary>
+		/// <param name="obj">The object to which the color is set.</param>
+		/// <returns>The foreground color for the tab bar.</returns>
 		public static Color GetTabBarTitleColor(BindableObject obj) => (Color)obj.GetValue(TabBarTitleColorProperty);
-		/// <include file="../../../docs/Microsoft.Maui.Controls/Shell.xml" path="//Member[@MemberName='SetTabBarTitleColor']/Docs/*" />
+
+		/// <summary>
+		/// Sets the title color for the tab bar. 
+		/// If the property is unset, the TitleColor property value will be used.
+		/// </summary>
+		/// <param name="obj">The object to which the color is set.</param>
+		/// <param name="value">The title color for the tab bar.</param>
 		public static void SetTabBarTitleColor(BindableObject obj, Color value) => obj.SetValue(TabBarTitleColorProperty, value);
 
-		/// <include file="../../../docs/Microsoft.Maui.Controls/Shell.xml" path="//Member[@MemberName='GetTabBarUnselectedColor']/Docs/*" />
+		/// <summary>
+		/// Gets the title color for the tab bar. 
+		/// </summary>
+		/// <param name="obj">The object to which the color is set.</param>
+		/// <returns>The title color for the tab bar.</returns>
 		public static Color GetTabBarUnselectedColor(BindableObject obj) => (Color)obj.GetValue(TabBarUnselectedColorProperty);
-		/// <include file="../../../docs/Microsoft.Maui.Controls/Shell.xml" path="//Member[@MemberName='SetTabBarUnselectedColor']/Docs/*" />
+
+		/// <summary>
+		/// Sets the unselected color for the tab bar. 
+		/// If the property is unset, the UnselectedColor property value is used.
+		/// </summary>
+		/// <param name="obj">The object to which the color is set.</param>
+		/// <param name="value">The unselected color for the tab bar.</param>
 		public static void SetTabBarUnselectedColor(BindableObject obj, Color value) => obj.SetValue(TabBarUnselectedColorProperty, value);
 
-		/// <include file="../../../docs/Microsoft.Maui.Controls/Shell.xml" path="//Member[@MemberName='GetTitleColor']/Docs/*" />
+		/// <summary>
+		/// Gets the unselected color for the tab bar. 
```

Comment:
This is wrong. The method is `GetTitleColor`

Reply:
Fixed